.env

```diff
@@ -0,0 +1,2 @@
+# pipenv configs
+PIPENV_VENV_IN_PROJECT=1
```

Comment:
I don't really have a preference myself, but I'm curious, is there any reason to overwrite the default?

Reply:
we always hat the venv inside the project directory, I just wanted to keep it as is. For me its easier that way to enter/delete the venv instead of having to looki it up with `pipenv --venv`.
Note: if you run `pipenv run python` you get a python console in the venv, but without arrow-up history functionality (which is broken for pipenv)

Reply:
Ok, I see.
For removing the venv `pipenv --rm` is handy.

Reply:
ah ya true, I forget about that. So should we keep it inside the project directory or not?

Reply:
Sure, like I said, was just curious. 🙂